adding process attributes to energy transformation #1543

viktorwichern opened the pull request:
I added two axioms to energy transformation. 
'energy transformation' 'has process attribute' some 'energy conversion efficiency'
'energy transformation' 'has process attribute' some power
Since parts of the relations were in oeo.shared, other parts in oeo.physical, I had to move power (OEO_00000333), energy conversion efficiency (OEO_00140049) and efficiency value (OEO_00140050) to oeo.shared. 

## Type of change (CHANGELOG.md)
-moved power, energy conversion efficiency and efficiency value to oeo.shared
-added axioms to energy transformation

### Updated
- Updated a definition [#](https://github.com/OpenEnergyPlatform/ontology/issues/)

## Workflow checklist

### Automation
Closes #1543 

### PR-Assignee
- [ ] 🐙 Follow the [Pull Request Workflow](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow)
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/ontology/blob/dev/CHANGELOG.md)
- [ ] 📙 Add #'s to `term tracker item`

### Reviewer
- [ ] 🐙 Follow the [Reviewer Guide](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow#reviewer-guide-check-changes-introduced-by-a-pull-request)
- [ ] 🐙 Provided feedback and show sufficient appreciation for the work done

stap-m commented:
> I added two axioms to energy transformation. 

Which ones? If you describe the header a little more detailed, it makes the review much easier.

Why did you update oeo.omn? It should be sufficient to update the physical and shared module.

viktorwichern commented:
> > I added two axioms to energy transformation.
> 
> Which ones? If you describe the header a little more detailed, it makes the review much easier.
> 
> Why did you update oeo.omn? It should be sufficient to update the physical and shared module.

I updated the description. 

I did not update oeo.omn myself as far as I am aware. I only moved from physical to shared. Should I delete the changes from oeo.omn?

viktorwichern commented:
I deleted the changes in oeo.omn.

stap-m reviewed (changes requested):
The moving of classes looks fine. But it seems that you forgot to add the axioms over all the moving 😀

viktorwichern commented:
> The moving of classes looks fine. But it seems that you forgot to add the axioms over all the moving 😬

Okay I think I know what went wrong, I opened the omn. file in Protege and added the axioms to the class without choosing the module, I thought that would happen automatically based on where the class is located. Instead it put the changes in the omn. file, which I had to revert of course. 

Since the new version is out, it would probably be correct to make a new pull request (again)?

stap-m commented:
You don't need a new PR. 
I resolved the merge conflicts. When you added all the changes, we'll take a look at CHANGELOG and move your changes to the new unreleased section (and add the section, in case it is not there yet).

stap-m reviewed (commented):
Ok 👍 
Now you have to fix the changelog, because your changes are still part of the old unreleased section, which is now released, i.e. release section 1.16.0.
You have to enter a new unreleased section (like e.g. @h-spinde did in #1633) and move your changelog comments there.

viktorwichern commented:
> Ok 👍
> Now you have to fix the changelog, because your changes are still part of the old unreleased section, which is now released, i.e. release section 1.16.0.
> You have to enter a new unreleased section (like e.g. @h-spinde did in #1633) and move your changelog comments there.

I updated the changelog, the unreleased section was already added.

stap-m commented:
Vielen Dank für Ihre Nachricht. Ich bin bis 25.08.2023 im Urlaub
      und werde Ihre Mail erst danach bearbeiten können.
    Thank you for your message. I am out of office until 25.08.2023.
    
    Mir freundlichen Grüßen
    Mirjam Stappel

stap-m commented on CHANGELOG.md:
```diff
@@ -34,6 +35,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 - climate scenario, climate system, energy scenario, carbon dioxide, energy system, supply system, emission scenario (#1615)
 - secondary energy production (#1619)
 - hydro energy, solar energy, wind energy (#1620)
+- energy transformation (#1625)
```
```suggestion
```
This is the already released section [1.16.0]. You did no changes there.

stap-m commented on CHANGELOG.md:
```diff
@@ -9,6 +9,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ### Changed
 - Update module files after saving with Protégé 5.6.1(#1628)
+- added process attributes to energy transformation (#1625)
```
```suggestion
- energy transformation (#1625)
```
We only write class or property names here. The changes can be tracked in detail via the PR number. This is why a good documentation of the PR description ois required.

stap-m replied:
Please see https://github.com/OpenEnergyPlatform/ontology/wiki/How-to-release-a-new-ontology-version for how to add an "unreleased section" to the changelog.

stap-m replied:
I fixed it.
